src/functionalTest/java/uk/gov/hmcts/reform/fact/functional/helpers/AssertionHelper.java

```diff
@@ -16,18 +16,37 @@ private AssertionHelper() {
 
     /**
      * Performs a GET request to the specified endpoint and validates the response.
-     * Asserts:
-     * - Response status is 200 OK
-     * - Content type is JSON
-     * - Response body is a non-null JSON array
      *
      * @param http the HTTP client
      * @param endpoint the endpoint path to test
+     * @param expectedStatus the expected HTTP status code
      */
-    public static void assertSuccessfulJsonArrayResponse(final HttpClient http, final String endpoint) {
+    public static void assertJsonArrayResponse(final HttpClient http,
+                                                final String endpoint,
+                                                final HttpStatus expectedStatus) {
         final Response response = http.doGet(endpoint);
-        assertThat(response.statusCode()).isEqualTo(OK.value());
+        assertThat(response.statusCode()).isEqualTo(expectedStatus.value());
         assertThat(response.contentType()).contains("json");
         assertThat(response.jsonPath().getList("$")).isNotNull();
     }
+
+    /**
+     * Performs a GET request to the specified endpoint and validates the response contains expected data.
+     * Useful for verifying endpoints are not misconfigured or returning the wrong data type.
+     *
+     * @param http the HTTP client
+     * @param endpoint the endpoint path to test
+     * @param expectedStatus the expected HTTP status code
+     * @param fieldName the field name to check in the first array element
+     */
+    public static void assertJsonArrayResponseHasField(final HttpClient http,
+                                                        final String endpoint,
+                                                        final HttpStatus expectedStatus,
+                                                        final String fieldName) {
+        final Response response = http.doGet(endpoint);
+        assertThat(response.statusCode()).isEqualTo(expectedStatus.value());
+        assertThat(response.contentType()).contains("json");
+        assertThat(response.jsonPath().getList("$")).isNotEmpty();
+        assertThat(response.jsonPath().getString("[0]." + fieldName)).isNotNull();
```

Comment:
its not actually checking the data returned, it could be good to include that as you know what the regionName could be for example and others as they are static lists